overlord/snapstate/handlers.go

```diff
@@ -253,7 +254,7 @@ func (m *SnapManager) installOneBaseOrRequired(st *state.State, snapName string,
 	}
 
 	// not installed, nor queued for install -> install it
-	ts, err := Install(st, snapName, &RevisionOptions{Channel: channel}, userID, Flags{RequireTypeBase: requireTypeBase})
+	ts, err := Install(context.TODO(), st, snapName, &RevisionOptions{Channel: channel}, userID, Flags{RequireTypeBase: requireTypeBase})
```

Comment:
how much bigger does this get if you make these take the context as well?

Reply:
I can do this in a followup. Note that right now we only use the context when we create the tasks (in the snapaction install API call to the store). So adding those will not add value right away.

Reply:
yeah, leave it for later